charts/dave/charts/eai/Chart.yaml

```
@@ -5,12 +5,12 @@
 maintainers:
-  - name: gislab-augsburg
-    email: martymcfly333@gmail.com
-  - name: klml
-    email: klml@muenchen.de
+- name: gislab-augsburg
+  email: martymcfly333@gmail.com
+- name: klml
+  email: klml@muenchen.de
 home: https://github.com/it-at-m/helm-charts/tree/main/charts/dave/charts/eai
-#icon: https://assets.muenchen.de/logos/itm/itm-logo-256.png
+#icon: https://assets.muenchen.de/logos/itm/itM_Basislogo_gelb_schwarz-256.png
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)